src/systemverilog/systemverilog.ts

```diff
@@ -393,10 +391,16 @@ export const language = <ILanguage>{
 		],
 
 		string: [
-			[/[^\\"]+/, 'string'],
```

Comment:
Do you need this line still to handle strings behind comments? ` // "not a string, actually a comment"